### ISSUE TITLE
Unrecognized SSL message, plaintext connection?

### ISSUE THREAD

**Issue by maticslaszlo** [apwelsh/hubitat#15]
My Hubitat can't communicate to Hue abot a few hours ago
wants to repair, but there is an error:
![image](https://user-images.githubusercontent.com/14295342/137643072-9d288bde-58d5-468a-b4eb-300a7905d648.png)

![image](https://user-images.githubusercontent.com/14295342/137643073-7300c802-6140-4950-bd23-8097aa4b2689.png)

LOGS:
```app:2842021-10-17 21:19:29.669 errororg.codehaus.groovy.runtime.InvokerInvocationException: javax.net.ssl.SSLException: Unrecognized SSL message, plaintext connection? (bridgeLinking)
app:2842021-10-17 21:19:29.620 infoAdding Bridge device with DNI: hue-ECB5FA2BA6C9
app:2842021-10-17 21:19:29.616 debugHub linking completed for Philips hue (2BA6C9)
app:2842021-10-17 21:19:27.543 debugSearching for Hub additions and updates
app:2842021-10-17 21:19:05.258 debugSearching for Hub additions and updates
app:2842021-10-17 21:19:03.539 errororg.codehaus.groovy.runtime.InvokerInvocationException: javax.net.ssl.SSLException: Unrecognized SSL message, plaintext connection? (bridgeLinking)
app:2842021-10-17 21:19:03.495 infoAdding Bridge device with DNI: hue-ECB5FA2BA6C9
app:2842021-10-17 21:19:03.491 debugHub linking completed for Philips hue (2BA6C9)
app:2842021-10-17 21:16:03.050 errororg.codehaus.groovy.runtime.InvokerInvocationException: javax.net.ssl.SSLException: Unrecognized SSL message, plaintext connection? (bridgeLinking)
app:2842021-10-17 21:16:02.992 infoAdding Bridge device with DNI: hue-ECB5FA2BA6C9
app:2842021-10-17 21:16:02.987 debugHub linking completed for Philips hue (2BA6C9)
app:2842021-10-17 21:16:00.710 debugSearching for Hub additions and updates
app:2842021-10-17 21:15:57.928 debugSearching for Hub additions and updates
app:2842021-10-17 21:15:55.238 debugSearching for Hub additions and updates
app:2842021-10-17 21:14:53.214 errororg.codehaus.groovy.runtime.InvokerInvocationException: javax.net.ssl.SSLException: Unrecognized SSL message, plaintext connection? (bridgeLinking)
app:2842021-10-17 21:14:53.144 infoAdding Bridge device with DNI: hue-ECB5FA2BA6C9
app:2842021-10-17 21:14:53.137 debugHub linking completed for Philips hue (2BA6C9)
app:2842021-10-17 21:14:46.597 debugSearching for Hub additions and updates
```

**Comment by apwelsh** [comment]
Thank you for reporting it here.  I went through a bunch of tests for adding new hubs from scratch to make sure all my new code was working.  Apparently, I missed re-testing when I converted the default protocol used to https (hue is pushing to convert to https).  I reverted the hub status refresh to plain text http, and this solved the problem for me.

Let me know if the current version that I just pushed works for you.
